css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,22 @@
+thead {
+  background-color: black;
+  color: white;
+}
+
+button {
+  width: 150px;
+  height: 50px;
+  font-size: 20px;
+  background-color: white;
+  border: solid;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+}
+
+.test {
```

Comment:
You should use a pseudoclass to accomplish this

Reply:
done.